diff --git a/src/hardware_interface.cpp b/src/hardware_interface.cpp
index 216d7d462..b500aadc7 100644
--- a/src/hardware_interface.cpp
+++ b/src/hardware_interface.cpp
@@ -45,3 +45,4 @@
+#include "ur_robot_driver/hardware_interface.hpp"
 #include "ur_client_library/exceptions.h"
 #include "ur_client_library/ur/tool_communication.h"
 #include "ur_client_library/ur/version_information.h"

diff --git a/include/ur_robot_driver/hardware_interface.hpp b/include/ur_robot_driver/hardware_interface.hpp
index 14bdd29ea..330e4161e 100644
--- a/include/ur_robot_driver/hardware_interface.hpp
+++ b/include/ur_robot_driver/hardware_interface.hpp
@@ -66,3 +66,5 @@
+typedef unsigned int uint;
+
 namespace ur_robot_driver
 {
 enum class PausingState


